proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -1147,3 +1147,17 @@ message UpdateActivityOptionsResponse {
     // Activity options after an update
     temporal.api.activity.v1.ActivityOptions activity_options = 1;
 }
+
+// (-- api-linter: core::0134::request-mask-required=disabled
+// (-- api-linter: core::0134::request-resource-required=disabled
+message PauseActivityRequest {
+    option (routing).workflow_id = "pause_request.workflow_id";
```

Comment:
Wondering if you had any issues using this new convention. Was it straightforward? Did you remember to add it in your first attempt or did you rely on an error message? If so, was the error message helpful?

Reply:
error message was clear.